cmd/soci-snapshotter-grpc/main.go

```diff
@@ -289,9 +290,10 @@ const (
 	dbMetadataType = "db"
 )
 
-func getMetadataStore(rootDir string, config config.Config) (metadata.Store, error) {
+func getMetadataStore(ctx context.Context, rootDir string, config config.Config) (metadata.Store, error) {
 	switch config.MetadataStore {
 	case "", dbMetadataType:
+		log.G(ctx).Debugf("initializing metadata db at %v (type: %v)", rootDir, config.MetadataStore)
```

Comment:
Let's use structured logging. See https://github.com/sirupsen/logrus?tab=readme-ov-file#fields

```suggestion
            log.G(ctx).WithField("root", rootDir).Debug("initializing metadata db")
```